### PR TITLE
fix: load policy by array to void problems caused by csv reader

### PR DESCRIPTION
**Pull request by fabian4**
Releated issue and pr
- https://github.com/mlsen/casbin-redis-adapter/issues/1
- https://github.com/casbin/casbin/issues/950
- https://github.com/casbin/redis-adapter/issues/23
- https://github.com/casbin/redis-adapter/pull/24

### CODE REVIEW

File: adapter.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (a *Adapter) loadPolicy(ctx context.Context, model model.Model, handler fun
 
 	// Parse the rules from Redis
 	for _, rule := range rules {
-		handler(strings.Split(rule, ","), model)
+		handler(strings.Split(rule, ", "), model)
```
Comment (mlsen) [inline adapter.go:66]: "test0,test1,  test2,     test3" is valid syntax, right? Your solution wouldn't work for that.
I would suggest splitting by ",", then loop through the slice and trim the spaces with strings.TrimSpace().

Sorry can't commit myself right now.

Comment (mlsen) [reply adapter.go:66]: > "test0,test1, test2, test3" is valid syntax, right? Your solution wouldn't work for that. I would suggest splitting by ",", then loop through the slice and trim the spaces with strings.TrimSpace().
> 
> Sorry can't commit myself right now.

Forget it, just checked it a bit more and it should be ok.

**Comment by mlsen** [comment]
Thanks a lot!

File: adapter.go (hunk 1 above)
Comment (fabian4) [reply adapter.go:66]: when you save the policy to redis by the rules below, you may split it by ', '
~~~go
func buildRuleStr(ptype string, rule []string) string {
	return ptype + ", " + util.ArrayToString(rule)
}
~~~

Comment (mlsen) [reply adapter.go:66]: Yep, exactly what I found out after checking the code for more than 1 minute. ;) Sorry!

**Pull request merged**